mlflow/utils/rest_utils.py

```diff
@@ -84,7 +84,7 @@ def _can_parse_as_json(string):
     try:
         json.loads(string)
         return True
-    except ValueError:
+    except (ValueError, TypeError):
```

Comment:
@andychow-db  Are these the only exception types guaranteed to be thrown by `json.loads`? It appears that the method documentation for `json.loads` refers to `JSONDecodeError` also being thrown: https://docs.python.org/3/library/json.html#json.JSONDecodeError. Is this a subclass of `TypeError`? If not, can we simply catch `Exception` (it's probably not the best practice, but I'd rather not miss a class of exception here)?

Reply:
JSONDecodeError is a subclass of ValueError [ref](https://docs.python.org/3/library/json.html#json.JSONDecodeError) and already handled with the existing code.

Reply:
To confirm, the TypeError is thrown when the argument passed to `json.loads` is `None` right?

Reply:
Agree with @dbczumar that it might be best to just catch a general `Exception` here, since the exception-throwing behavior of `json.loads` [isn't well-documented](https://docs.python.org/3/library/json.html#json.loads). It seems more likely that we'd miss a class of exception inadvertently (by not catching `Exception`) than that we'd accidentally swallow an important exception (if we were to catch `Exception`)

Reply:
Agreed, I've replaced the explicit error types with the general Exception type.